packages/sdks/web-component/src/lib/descope-wc/DescopeWc.ts

```diff
@@ -262,11 +298,25 @@ class DescopeWc extends BaseDescopeWc {
     return undefined;
   }
 
+  #subscribeStepState() {
+    this.stepState?.subscribe(
+      this.onStepChange.bind(this),
+      ({
+        screenState: { errorText, errorType, ...screenState } = {},
+        ...state
+      }) => ({ ...state, screenState }),
+    );
+    this.stepState?.subscribe(this.#handleGlobalErrors.bind(this), (state) => ({
```

Comment:
if I understand correctly, the first `onStepChange` will remove the errorText,errorType from state, and the second will assign them again, so order matters, here... is that right? or am I missing something?

Reply:
No, I added the option to subscribe to a partial state by providing a selector 
In the first subscription, we exclude the errors, and in the second one, we exclude the rest of the state